acceptance-tests/src/test/java/uk/gov/di/ipv/cri/passport/acceptance_tests/pages/PagesFromIpvCoreRepo/FraudCheckStubPage.java

```diff
@@ -27,27 +27,6 @@ public FraudCheckStubPage() {
     @FindBy(xpath = "//input[@name='submit']")
     public WebElement SubmitDataAndGenerateAuthCode;
 
-    @FindBy(id = "test_data")
-    public WebElement SelectCRISubData;
-
-    @FindBy(id = "fraud")
-    public WebElement GPG45Fraud;
-
-    @FindBy(id = "ci")
-    public WebElement Contraindicators;
-
-    @FindBy(id = "endpoint")
-    public WebElement Authorization;
-
-    @FindBy(id = "requested_oauth_error_endpoint_2")
-    public WebElement AccessToken;
-
-    @FindBy(id = "requested_oauth_error")
-    public WebElement OAuthError;
-
-    @FindBy(id = "requested_oauth_error_description")
-    public WebElement OAuthErrorDescription;
-
     public static void enterFraudDetails() {
```

Comment:
Are we actually using this page?

Reply:
Hi @gds-jbentham, yes, these are being used by the accessibilty tests which we have copied over from the ipv-core-tests repo. These are currently not working but will be fixed as part of the ticket https://govukverify.atlassian.net/browse/LIME-625